secrets.ts

```diff
@@ -252,7 +252,20 @@ export const TELEGRAM_API_HASH = process.env.TELEGRAM_API_HASH || '';
  * The production environement.
  * Fallback: an empty string if not defined in the environment.
  */
-export const PRODUCTION_ENV = process.env.PRODUCTION_ENV || '';
+export const PRODUCTION_ENV =
+  /^true$/i.test(process.env.PRODUCTION_ENV!) || false;
+
+/**
+ * The staging environement.
+ * Fallback: an empty string if not defined in the environment.
+ */
+export const STAGING_ENV = /^true$/i.test(process.env.STAGING_ENV!) || false;
+
+/**
+ * The test environement.
+ * Fallback: an empty string if not defined in the environment.
+ */
+export const TEST_ENV = /^true$/i.test(process.env.TEST_ENV!) || false;
```

Comment:
I think we should use `NODE_ENV` to determine environment as that's more standard and easier to understand. Also we should not detect staging here.

```suggestion
export const PRODUCTION_ENV = process.env.NODE_ENV === 'production';

/**
 * The test environement.
 * Fallback: an empty string if not defined in the environment.
 */
export const TEST_ENV = !PRODUCTION_ENV;
```